src/jit/lsra.cpp

```diff
@@ -4148,6 +4161,14 @@ void LinearScan::buildRefPositionsForNode(GenTree*                  tree,
             useCandidates  = allRegs(registerType);
         }
 
+#ifdef ARM_SOFTFP
```

Comment:
This part is kind of pair with lin 421 at laraarmarch.cpp, but I am not sure about this change. @caroleidt this is the part that I was wondering from the our last conversation #12293.